hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenRevokeKycFromAccountHandler.java

```diff
@@ -40,27 +46,57 @@ public TokenRevokeKycFromAccountHandler() {
         // Exists for injection
     }
 
+    /**
+     * This method is called during the pre-handle workflow.
+     *
+     * @param context the {@link PreHandleContext} which collects all information
+     * @throws PreCheckException    for invalid tokens or if the token has no KYC key
+     * @throws NullPointerException if one of the arguments is {@code null}
+     */
     @Override
     public void preHandle(@NonNull final PreHandleContext context) throws PreCheckException {
         requireNonNull(context);
         final var op = context.body().tokenRevokeKycOrThrow();
         final var tokenStore = context.createStore(ReadableTokenStore.class);
         final var tokenMeta = tokenStore.getTokenMeta(op.tokenOrElse(TokenID.DEFAULT));
-        if (tokenMeta == null) throw new PreCheckException(INVALID_TOKEN_ID);
+        if (tokenMeta == null) throw new PreCheckException(ResponseCodeEnum.INVALID_TOKEN_ID);
```

Comment:
FYI -- in a lot of other places I've seen us switch from ResponseCodeEnum.FOO to just using FOO.

Reply:
Good to know. Guess I've been hitting the other option in the intellij dropdown 😂